src/generate/codemodel/Command.ts

```diff
@@ -25,8 +25,8 @@ export interface CommandModel {
     Command_Mode: string;
     Command_Type: string;
     Command_GenericSetterArgName: string;
-    // Command_Features: [string, string];
-    // Command_Imports: [string, string[]];
+    Command_Features: Record<string, any>;
```

Comment:
the value type of Features is a string? wondering is it necessary to use 'any'?

Reply:
no. it's a key value dict

Reply:
Not exactly. I don't want to rule out the possiblity of other complex cases. For example 
```
directive:
    command: some-command-name
feature:
   some_feature: 
       - a
       - 'b'
   some_feature1:
      a1: a2
      'b1': 'b2'
```

Reply:
I see. Will it be related to below code? looks currently only string type is handled for "feature"
![image](https://user-images.githubusercontent.com/59815250/115174714-56006000-a0fc-11eb-917a-93d2e12fd225.png)

Reply:
Actually this works for both string or number type, I need to add logic about the complex scenario I mentioned above

Reply:
yes, if it's also supposed to by a number, it will be better to use "prop: string|number"

Reply:
changed,